package.json

```diff
@@ -56,14 +56,14 @@
     "slick-carousel": "1.6.0",
     "style-loader": "2.0.0",
     "stylelint": "13.13.1",
-    "stylelint-declaration-use-variable": "1.7.2",
+    "stylelint-declaration-use-variable": "^1.7.3",
     "svgo": "2.3.0",
-    "vue": "2.6.12",
+    "vue": "^2.6.12",
     "vue-async-computed": "3.9.0",
     "vue-multiselect": "2.1.6",
     "vue-template-compiler": "2.6.12",
-    "webpack": "4.46.0",
-    "webpack-cli": "4.6.0"
+    "webpack": "^4.46.0",
```

Comment:
Thanks for submitting the patch! We pin version numbers in the codebase to ensure that we all use the same working environment. Can you adjust this patchset so that you do not use "^" character. e.g. `"webpack": "4.46.0",`

Note: We do have a bot which does these upgrades for us periodically.

Reply:
Doesn't package-lock.json ensure that?